python/practices/practice-3/src/path1/required_letter_word_counter.py

```diff
@@ -0,0 +1,58 @@
+"""
+Check and print all word that uses all the letters in a string
+"""
+
+
+import os
+
+
+def uses_all(word, required_letters):
+    """
+    Check if a word contains all the required letters.
+
+    Parameters:
+        - word (str): The word to check.
+        - required_letters (str): A string containing the letters that must be
+        present in 'word'.
+
+    Returns:
+        - bool: True if 'word' contains all the required letters, False
+        otherwise.
+    """
+
+    for letter in required_letters:
+        if letter not in word:
+            return False
+
+    return True
+
+
+def count_words_vowel(required_letters):
```

Comment:
I think `count_words` is a good name.

Reply:
I think you should refactor this function, it should accept a list of words and required letters.